templates/taxi/driver_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Driver List
+      <a href="{% url 'taxi:driver-create' %}">Create</a>
```

Comment:
Consider adding pagination to the driver list if the number of drivers is large. This can improve user experience by making the list more manageable and faster to load.